src/omv/ports/stm32/sensor.c

```diff
@@ -1139,10 +1181,10 @@ static void sensor_check_buffsize()
     if (sensor.pixformat == PIXFORMAT_RGB565) {
         // Switch to bayer for the quick 2x savings.
         sensor_set_pixformat(PIXFORMAT_BAYER);
-        bpp = 1;
+        bpp = sizeof(uint8_t);
```

Comment:
```suggestion
        bpp = 1;
```

Reply:
I prefer the sizeo(uint8_t). It infers intent.

Reply:
Done